pkg/ssh/connection_golang.go

```diff
@@ -64,6 +64,23 @@ func golangConnectionCreate(options ConnectionCreateOptions) error {
 	} else {
 		cfg.Engine.ServiceDestinations[options.Name] = *dst
 	}
+
+	// Create or update an existing farm with the connection being added
+	if options.Farm != "" {
+		if cfg.Farms.List == nil {
```

Comment:
I think that `cfg.Farms.List` should never be `nil`.  That's a guarantee we could/should give to callers (Podman) to simplify call sites.